contrib/completion/bash/docker

```diff
@@ -5485,10 +5485,11 @@ _docker_wait() {
 	_docker_container_wait
 }
 
+COMPOSE_PLUGIN_PATH=$(docker info --format '{{json .ClientInfo.Plugins}}' | jq '.[] | select(.Name=="compose") | .Path')
```

Comment:
Sadly, we cannot rely on jq being installed.

One solution could be
```bash
docker info --format '{{json .ClientInfo.Plugins}}' | sed -n 's/.*"Path":"\([^"]\+docker-compose\)".*/\1/p'
```

If we use `docker info` we should also ensure that it is not being called multiple times during one completion.
I can take care of this based on the existing [basic result caching](https://github.com/docker/cli/blob/5f07d7d5a12423c0bc1fb507f4d006ad0cdfef42/contrib/completion/bash/docker#L603-L608).

We can avoid the whole issue by checking hard coded well known install locations, i.e. `/usr/libexec/docker/cli-plugins/` and `~/.docker/cli-plugins/`.

Reply:
Cool! It's replaced with `sed`!

The problem with a hardcoded path solution is that new locations can be configured in `~/.docker/config.json`.

Reply:
> The problem with a hardcoded path solution is that new locations can be configured in ~/.docker/config.json.

Good point. I did not know this.